src/ui/components/shared/SharingModal/SharingModal.module.css

```diff
@@ -0,0 +1,37 @@
+.copy-link input[type="text"] {
```

Comment:
This looks like a mistake. Styles in CSS module files (`*.module.css`) will have their names mangled during build to avoid naming conflicts. That's one of the nice things about CSS styles. The names get matched up because you generally refer to them like so:

```tsx
// MyComponent.tsx
import styles from "./MyComponent.module.css";

<div className={styles.Example}>...</div>
```

Generally they shouldn't have hyphens in them because then you have to refer to them like this:
```tsx
<div className={styles["copy-link"}>...</div>
```

In this case, it looks like the style is being referred to like this:
```tsx
<div className="copy-link relative flex flex-shrink-0 flex-col items-center">
```

This won't actually do anything, because the "copy-link" style name in this module will be converted to something random.

Also, looking at the `ReplayLink` file where this is being used– there doesn't seem to be any scenario in which there's an `<input>` inside of a "copy-link" styled component anyway?

Reply:
Good eye. This was from @jaril about a year ago, and I'm seeing the same things you are. But I do remember this functionality existing somewhere in the code so I'm reluctant to remove it in case I blow something up.

Because this already exists on production, I'd propose a separate task to investigate this and possibly remove those lines of code if they're truly unused.